include/libxnvme_nvm.h

```diff
@@ -111,6 +111,22 @@ xnvme_nvm_scopy(struct xnvme_cmd_ctx *ctx, uint32_t nsid, uint64_t sdlba,
 		struct xnvme_spec_nvm_scopy_fmt_zero *ranges, uint8_t nr,
 		enum xnvme_nvm_scopy_fmt copy_fmt);
 
+/**
+ * Deallocate or hint at read/write usage of a range.
+ *
+ * @param ctx Pointer to command context (::xnvme_cmd_ctx)
+ * @param nsid Namespace Identifier
+ * @param nr Number of ranges in the given ranges-buffer, zero-based value
+ * @param ad The NVM subsystem may deallocate all provided ranges
+ * @param idw The dataset should be optimized for write access as an integral unit
+ * @param idr The dataset should be optimized for read access as an integral unit
```

Comment:
This seems to be missing a comment of ``dsm_range``

Reply:
Done